robottelo/content_info.py

```diff
@@ -0,0 +1,53 @@
+"""Module that gather several informations about host"""
+from robottelo import ssh
+from robottelo.cli.base import CLIReturnCodeError
+
+
+def get_repo_files(repo_path, extension='rpm', hostname=None):
```

Comment:
Looking for the callers for these functions - why aren't they ContentHost class methods?

Reply:
This method is really only used by one file, so technically could live there. I wouldn't want it on the main ContentHost class yet, but maybe after a helpers consolidation.